core/src/main/java/com/exadel/aem/toolkit/core/injectors/utils/CastResult.java

```diff
@@ -0,0 +1,28 @@
+package com.exadel.aem.toolkit.core.injectors.utils;
+
+public class CastResult {
+
+    private Object value;
+    private boolean isFallback;
+
+    public CastResult(Object value, boolean isFallback) {
```

Comment:
I think the constructor could be package-private, and also you don't seem to need setters at all

Reply:
fixed